catroid/src/main/res/values/strings.xml

```diff
@@ -1167,15 +1164,15 @@
     <string name="brick_stop_script">Stop</string>
     <string name="brick_stop_this_script">this script</string>
     <string name="brick_stop_all_scripts">all scripts</string>
-    <string name="brick_stop_other_scripts">other scripts of this object</string>
+    <string name="brick_stop_other_scripts">other scripts of this actor or object</string>
 
 
     <plurals name="second_plural">
         <item quantity="one">"second "</item>
         <item quantity="other">"seconds "</item>
     </plurals>
 
-    <string name="brick_collision_receive">When physical collision with</string>
+    <string name="brick_collision_receive">When collision under gravity with</string>
     <string name="collision_with_anything">anything</string>
```

Comment:
```suggestion
    <string name="collision_with_anything">any edge or object of motion type \'bouncing\'</string>
```
Clearer and more precise than what we have now, which is understandably poorly understood by our users as it is.